backend/src/main/java/trn/logistics/knapsack/dto/AlgorithmPair.java

```diff
@@ -0,0 +1,14 @@
+package trn.logistics.knapsack.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+public class AlgorithmPair {
```

Comment:
```suggestion
/**
* This is a helper class to return the result of the knapsack algorithm
*/
public class AlgorithmPair {
```
Maybe its helpful to add a comment